ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -85,10 +92,46 @@ export default class ExperimentalTab extends PureComponent {
     );
   }
 
+  renderOpenSeaEnabledToggle() {
+    const { t } = this.context;
+    const { openSeaEnabled, setOpenSeaEnabled } = this.props;
+
+    return (
+      <div className="settings-page__content-row">
+        <div className="settings-page__content-item">
+          <span>{t('enableOpenSeaAPI')}</span>
+          <div className="settings-page__content-description">
+            {t('enableOpenSeaAPIDescription')}
+          </div>
+        </div>
+        <div className="settings-page__content-item">
+          <div className="settings-page__content-item-col">
+            <ToggleButton
+              value={openSeaEnabled}
+              onToggle={(value) => {
+                this.context.metricsEvent({
+                  eventOpts: {
+                    category: 'Settings',
+                    action: 'Enabled/Disable OpenSea',
+                    name: 'Enabled/Disable OpenSea',
+                  },
+                });
+                setOpenSeaEnabled(!value);
+              }}
+              offLabel={t('off')}
+              onLabel={t('on')}
+            />
+          </div>
+        </div>
+      </div>
+    );
+  }
+
   render() {
     return (
       <div className="settings-page__body">
         {this.renderTokenDetectionToggle()}
+        {this.renderOpenSeaEnabledToggle()}
         {this.renderCollectibleDetectionToggle()}
       </div>
     );
```

Comment:
Will be useful to have some test coverage of UI code also, may be in a following PR.